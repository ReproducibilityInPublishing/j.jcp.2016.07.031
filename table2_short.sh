echo "AIMGM" > table2.csv

./example1 -l 2 -N 8192 -a 0.01 2>/dev/null >> table2.csv
./example1 -l 3 -N 8192 -a 0.01 2>/dev/null >> table2.csv
./example1 -l 4 -N 8192 -a 0.01 2>/dev/null >> table2.csv
./example1 -l 5 -N 8192 -a 0.01 2>/dev/null >> table2.csv
./example1 -l 6 -N 8192 -a 0.01 2>/dev/null >> table2.csv
./example1 -l 7 -N 8192 -a 0.01 2>/dev/null >> table2.csv

echo "BD-ADI" >> table2.csv

./example1_BDADI -l 2 -N 8192 -a 0.01 2>/dev/null >> table2.csv
./example1_BDADI -l 3 -N 8192 -a 0.01 2>/dev/null >> table2.csv
./example1_BDADI -l 4 -N 8192 -a 0.01 2>/dev/null >> table2.csv
./example1_BDADI -l 5 -N 8192 -a 0.01 2>/dev/null >> table2.csv
./example1_BDADI -l 6 -N 8192 -a 0.01 2>/dev/null >> table2.csv
#./example1_BDADI -l 7 -N 8192 -a 0.01 2>/dev/null >> table2.csv
