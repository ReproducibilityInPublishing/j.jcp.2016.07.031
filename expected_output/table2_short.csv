AIMGM
0,0,3.14159,3.14159,0.5,5e-09,1e-08,4,0.01,8192,1,0.075901,3.454477e-02
0,0,3.14159,3.14159,0.5,5e-09,1e-08,8,0.01,8192,3,0.232389,8.549911e-03
0,0,3.14159,3.14159,0.5,5e-09,1e-08,16,0.01,8192,4,1.1847,2.132079e-03
BD-ADI
0,0,3.14159,3.14159,0.5,5e-09,1e-08,4,0.01,8192,0,0.557702,3.447405e-02
0,0,3.14159,3.14159,0.5,5e-09,1e-08,8,0.01,8192,0,4.25719,8.477281e-03
0,0,3.14159,3.14159,0.5,5e-09,1e-08,16,0.01,8192,0,56.2289,2.058973e-03
